Freshly created container with MC 1.20.6 results in failure to open ViaVersion, zip END header not found

## Context

On startup, the server log shows the error 'java.lang.RuntimeException: Failed to open plugin jar plugins/ViaVersion.jar' & `Caused by: java.util.zip.ZipException: zip END header not found`. Other plugins don't load or even show a failure message in the log. Removing and reinstalling ViaVersion did not solve the issue.

The error started happening this morning 2024-09-04, and the server was running fine using this image before that point. Since there hasn't been a new release of LJMGF, I'm guessing there was an update in one of the dependencies that bricked it.

I reproduced the error by doing `docker compose up -d` with the below docker-compose.yaml in an empty volume with no plugins or world data added. I tried with and without NoPermCheck and got the same result.

## docker-compose.yaml

```yaml
services:
  manatee-mc:
    image: 05jchambers/legendary-minecraft-geyser-floodgate:latest
    container_name: manatee-mc
    environment:
      Version: 1.20.6
      MaxMemory: 8192
      TZ: America/Los_Angeles
      # NoPermCheck: Y
    volumes:
      - ./lmgf:/minecraft
    ports:
      - "25565:25565"
      - "19132:19132/udp"
      - "19132:19132"
      - "8123:8123"
    restart: always
  manatee-rcon:
    image: itzg/rcon
    container_name: manatee-rcon
    restart: always
    environment:
      TZ: America/Los_Angeles
      RWA_USERNAME: *****
      RWA_PASSWORD: *****
      RWA_ADMIN: "TRUE"
      # is referring to the service name 'mc' declared below
      RWA_RCON_HOST: manatee-mc
      # needs to match the password configured for the container, see RCON_PASSWORD below
      RWA_RCON_PASSWORD: "*****"
    ports:
      - "4326:4326"
      - "4327:4327"
```

## Log

```
## Log2024-09-04 19:41:52 Paper Minecraft Java Server Docker + Geyser/Floodgate script by James A. Chambers
2024-09-04 19:41:52 Latest version always at https://github.com/TheRemote/Legendary-Java-Minecraft-Geyser-Floodgate
2024-09-04 19:41:52 Don't forget to set up port forwarding on your router!  The default port is 25565 and the Bedrock port is 19132
2024-09-04 19:41:52 Port used: 25565
2024-09-04 19:41:52 Bedrock port used: 19132
2024-09-04 19:41:52 Taking ownership of all server files/folders in /minecraft...
2024-09-04 19:41:52 Complete
2024-09-04 19:41:52 Updating to most recent paperclip version ...
2024-09-04 19:41:54 Latest paperclip build found: 149
2024-09-04 19:41:56 Updating Floodgate...
2024-09-04 19:41:57 Updating Geyser...
2024-09-04 19:41:54   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
2024-09-04 19:41:54                                  Dload  Upload   Total   Spent    Left  Speed
2024-09-04 19:41:56 
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 21 43.6M   21 9550k    0     0  15.3M      0  0:00:02 --:--:--  0:00:02 15.3M
 60 43.6M   60 26.2M    0     0  16.3M      0  0:00:02  0:00:01  0:00:01 16.3M
 97 43.6M   97 42.7M    0     0  16.3M      0  0:00:02  0:00:02 --:--:-- 16.3M
0100-01-01 00:00:00 43.6M  100 43.6M    0     0  16.3M      0  0:00:02  0:00:02 --:--:-- 16.3M
2024-09-04 19:41:56   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
2024-09-04 19:41:56                                  Dload  Upload   Total   Spent    Left  Speed
2024-09-04 19:41:57 
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
2024-09-04 19:41:57 
0100-01-01 00:00:00 10.9M  100 10.9M    0     0  12.1M      0 --:--:-- --:--:-- --:--:-- 12.1M
2024-09-04 19:41:57   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
2024-09-04 19:41:57                                  Dload  Upload   Total   Spent    Left  Speed
2024-09-04 19:41:58 
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
2024-09-04 19:41:59 
 40 20.7M   40 8638k    0     0  8454k      0  0:00:02  0:00:01  0:00:01 8454k
0100-01-01 00:00:00 20.7M  100 20.7M    0     0  11.5M      0  0:00:01  0:00:01 --:--:-- 15.9M
2024-09-04 19:42:00 md5sum: plugins/ViaVersion.jar: No such file or directory
2024-09-04 19:42:00 Updating ViaVersion...
2024-09-04 19:42:00   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
2024-09-04 19:42:00                                  Dload  Upload   Total   Spent    Left  Speed
2024-09-04 19:42:01 
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
0100-01-01 00:00:00  9737    0  9737    0     0  18094      0 --:--:-- --:--:-- --:--:-- 18064
2024-09-04 19:42:01 Starting Minecraft server...
2024-09-04 19:42:01 Downloading mojang_1.20.6.jar
2024-09-04 19:42:13 Applying patches
2024-09-04 19:42:17 Starting org.bukkit.craftbukkit.Main
2024-09-04 19:42:18 System Info: Java 21 (OpenJDK 64-Bit Server VM 21.0.3+9-Ubuntu-1ubuntu1) Host: Linux 5.15.153.1-microsoft-standard-WSL2 (amd64)
2024-09-04 19:42:18 Loading libraries, please wait...
2024-09-04 19:42:18 WARN StatusConsoleListener Advanced terminal features are not available in this environment
2024-09-04 19:42:20 [19:42:20 INFO]: [ReobfServer] Remapping server...
2024-09-04 19:42:21 [19:42:21 ERROR]: [EntrypointUtil] Failed to open plugin jar plugins/ViaVersion.jar
2024-09-04 19:42:21 java.lang.RuntimeException: Failed to open plugin jar plugins/ViaVersion.jar
2024-09-04 19:42:21     at io.papermc.paper.pluginremap.PluginRemapper.remap(PluginRemapper.java:339) ~[paper-1.20.6.jar:1.20.6-149-6e71f41]
2024-09-04 19:42:21     at io.papermc.paper.pluginremap.PluginRemapper.remapPlugin(PluginRemapper.java:266) ~[paper-1.20.6.jar:1.20.6-149-6e71f41]
2024-09-04 19:42:21     at io.papermc.paper.pluginremap.PluginRemapper.rewritePluginDirectory(PluginRemapper.java:204) ~[paper-1.20.6.jar:1.20.6-149-6e71f41]
2024-09-04 19:42:21     at io.papermc.paper.plugin.provider.source.DirectoryProviderSource.prepareContext(DirectoryProviderSource.java:42) ~[paper-1.20.6.jar:1.20.6-149-6e71f41]
2024-09-04 19:42:21     at io.papermc.paper.plugin.provider.source.DirectoryProviderSource.prepareContext(DirectoryProviderSource.java:17) ~[paper-1.20.6.jar:1.20.6-149-6e71f41]
2024-09-04 19:42:21     at io.papermc.paper.plugin.util.EntrypointUtil.registerProvidersFromSource(EntrypointUtil.java:14) ~[paper-1.20.6.jar:1.20.6-149-6e71f41]
2024-09-04 19:42:21     at io.papermc.paper.plugin.PluginInitializerManager.load(PluginInitializerManager.java:109) ~[paper-1.20.6.jar:1.20.6-149-6e71f41]
2024-09-04 19:42:21     at net.minecraft.server.Main.main(Main.java:122) ~[paper-1.20.6.jar:1.20.6-149-6e71f41]
2024-09-04 19:42:21     at org.bukkit.craftbukkit.Main.main(Main.java:328) ~[paper-1.20.6.jar:1.20.6-149-6e71f41]
2024-09-04 19:42:21     at io.papermc.paperclip.Paperclip.lambda$main$0(Paperclip.java:42) ~[app:?]
2024-09-04 19:42:21     at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
2024-09-04 19:42:21 Caused by: java.util.zip.ZipException: zip END header not found
2024-09-04 19:42:21     at jdk.zipfs@21.0.3/jdk.nio.zipfs.ZipFileSystem.findEND(ZipFileSystem.java:1320) ~[jdk.zipfs:?]
2024-09-04 19:42:21     at jdk.zipfs@21.0.3/jdk.nio.zipfs.ZipFileSystem.initCEN(ZipFileSystem.java:1541) ~[jdk.zipfs:?]
2024-09-04 19:42:21     at jdk.zipfs@21.0.3/jdk.nio.zipfs.ZipFileSystem.<init>(ZipFileSystem.java:179) ~[jdk.zipfs:?]
2024-09-04 19:42:21     at jdk.zipfs@21.0.3/jdk.nio.zipfs.ZipFileSystemProvider.getZipFileSystem(ZipFileSystemProvider.java:125) ~[jdk.zipfs:?]
2024-09-04 19:42:21     at jdk.zipfs@21.0.3/jdk.nio.zipfs.ZipFileSystemProvider.newFileSystem(ZipFileSystemProvider.java:120) ~[jdk.zipfs:?]
2024-09-04 19:42:21     at java.base/java.nio.file.FileSystems.newFileSystem(FileSystems.java:528) ~[?:?]
2024-09-04 19:42:21     at java.base/java.nio.file.FileSystems.newFileSystem(FileSystems.java:440) ~[?:?]
2024-09-04 19:42:21     at io.papermc.paper.pluginremap.PluginRemapper.remap(PluginRemapper.java:290) ~[paper-1.20.6.jar:1.20.6-149-6e71f41]
2024-09-04 19:42:21     ... 10 more
2024-09-04 19:42:27 [19:42:27 INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
2024-09-04 19:42:27 [19:42:27 INFO]: Found new data pack file/bukkit, loading it automatically
2024-09-04 19:42:27 [19:42:27 INFO]: Found new data pack paper, loading it automatically
2024-09-04 19:42:29 [19:42:29 INFO]: No existing world data, creating new world
2024-09-04 19:42:29 [19:42:29 WARN]: Failed to parse level-type default, defaulting to minecraft:normal
2024-09-04 19:42:32 [19:42:32 INFO]: [ReobfServer] Done remapping server in 11692ms.
2024-09-04 19:42:32 [19:42:32 INFO]: [PluginRemapper] Remapping plugin 'plugins/Floodgate-Spigot.jar'...
2024-09-04 19:42:32 [19:42:32 INFO]: Loaded 1175 recipes
2024-09-04 19:42:32 [19:42:32 INFO]: Loaded 1276 advancements
2024-09-04 19:42:32 [19:42:32 INFO]: Starting minecraft server version 1.20.6
2024-09-04 19:42:32 [19:42:32 WARN]: ****************************
2024-09-04 19:42:32 [19:42:32 WARN]: YOU ARE RUNNING THIS SERVER AS AN ADMINISTRATIVE OR ROOT USER. THIS IS NOT ADVISED.
2024-09-04 19:42:32 [19:42:32 WARN]: YOU ARE OPENING YOURSELF UP TO POTENTIAL RISKS WHEN DOING THIS.
2024-09-04 19:42:32 [19:42:32 WARN]: FOR MORE INFORMATION, SEE https://madelinemiller.dev/blog/root-minecraft-server/
2024-09-04 19:42:32 [19:42:32 WARN]: ****************************
2024-09-04 19:42:32 [19:42:32 INFO]: Loading properties
2024-09-04 19:42:32 [19:42:32 INFO]: This server is running Paper version 1.20.6-149-ver/1.20.6@6e71f41 (2024-08-24T09:37:06Z) (Implementing API version 1.20.6-R0.1-SNAPSHOT)
2024-09-04 19:42:33 [19:42:33 INFO]: Server Ping Player Sample Count: 12
2024-09-04 19:42:33 [19:42:33 INFO]: Using 4 threads for Netty based IO
2024-09-04 19:42:34 [19:42:34 WARN]: [!] The timings profiler has been enabled but has been scheduled for removal from Paper in the future.
2024-09-04 19:42:34     We recommend installing the spark profiler as a replacement: https://spark.lucko.me/
2024-09-04 19:42:34     For more information please visit: https://github.com/PaperMC/Paper/issues/8948
2024-09-04 19:42:34 [19:42:34 INFO]: [ChunkTaskScheduler] Chunk system is using 1 I/O threads, 4 worker threads, and gen parallelism of 4 threads
2024-09-04 19:42:34 [19:42:34 INFO]: Default game type: SURVIVAL
2024-09-04 19:42:34 [19:42:34 INFO]: Generating keypair
2024-09-04 19:42:35 [19:42:35 INFO]: Starting Minecraft server on *:25565
2024-09-04 19:42:35 [19:42:35 INFO]: Using epoll channel type
2024-09-04 19:42:35 [19:42:35 INFO]: Paper: Using libdeflate (Linux x86_64) compression from Velocity.
2024-09-04 19:42:35 [19:42:35 INFO]: Paper: Using OpenSSL 3.0.x (Linux x86_64) cipher from Velocity.
2024-09-04 19:42:35 [19:42:35 INFO]: Preparing level "world"
2024-09-04 19:42:35 [19:42:35 WARN]: Failed to parse level-type default, defaulting to minecraft:normal
2024-09-04 19:42:37 [19:42:37 INFO]: [PluginRemapper] Done remapping plugin 'plugins/Floodgate-Spigot.jar' in 5405ms.
2024-09-04 19:42:38 [19:42:38 WARN]: Failed to parse level-type default, defaulting to minecraft:normal
2024-09-04 19:42:39 [19:42:39 WARN]: Failed to parse level-type default, defaulting to minecraft:normal
2024-09-04 19:42:39 [19:42:39 INFO]: Preparing start region for dimension minecraft:overworld
2024-09-04 19:42:41 [19:42:41 INFO]: Time elapsed: 1713 ms
2024-09-04 19:42:41 [19:42:41 INFO]: Preparing start region for dimension minecraft:the_nether
2024-09-04 19:42:41 [19:42:41 INFO]: Time elapsed: 638 ms
2024-09-04 19:42:41 [19:42:41 INFO]: Preparing start region for dimension minecraft:the_end
2024-09-04 19:42:41 [19:42:41 INFO]: Time elapsed: 235 ms
2024-09-04 19:42:42 [19:42:42 INFO]: Running delayed init tasks
2024-09-04 19:42:42 [19:42:42 INFO]: Done (9.334s)! For help, type "help"
2024-09-04 19:42:42 [19:42:42 INFO]: Timings Reset
```

--- a/start.sh
+++ b/start.sh
@@ -172,7 +172,7 @@ else
 
     if [ -z "$NoViaVersion" ]; then
         # Update ViaVersion if new version is available
-        ViaVersionVersion=$(curl --no-progress-meter -k -L -H "Accept-Encoding: identity" -H "Accept-Language: en" -L -A "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4.212 Safari/537.36" https://ci.viaversion.com/job/ViaVersion/lastBuild/artifact/build/libs/ | grep 'href="ViaVersion' | cut -d'"' -f2)
+        ViaVersionVersion=$(curl --no-progress-meter -k -L -H "Accept-Encoding: identity" -H "Accept-Language: en" -L -A "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4.212 Safari/537.36" https://ci.viaversion.com/job/ViaVersion/lastBuild/artifact/build/libs/ | grep -P '(?<=href=")ViaVersion[^"]+' -o --max-count=1 | head -n1)
         if [ -n "$ViaVersionVersion" ]; then
             ViaVersionMD5=$(curl --no-progress-meter -k -L -H "Accept-Encoding: identity" -H "Accept-Language: en" -L -A "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4.212 Safari/537.36" "https://ci.viaversion.com/job/ViaVersion/lastBuild/artifact/build/libs/$ViaVersionVersion/*fingerprint*/" | grep breadcrumbs | cut -d'_' -f24- | cut -d'<' -f2 | cut -d'>' -f2)
             if [ -n "$ViaVersionMD5" ]; then

